### DISCUSSION TITLE
I Have over 952k sats stuck when closing a channel

### DISCUSSION THREAD

**Discussion by deserteagle2303** [lightningnetwork/lnd#7190] [Troubleshooting]
Hello please find below 2 transactions regarding closing channels that have been in limbo since October 7th, 2022.
The amounts stuck in Limbo is 952,210 sats.

Please help to “unstick” them.

Thank you

Total balance in Limbo: 
Total Limbo Balance: 952,210 Sats

Wait Closing Channel Information
X

Transaction ID: 0af13ff945a52682c2271d754cfa1970f2aa8fb49d5088b2c3e236b9b67563f2

Channel Point: 717143d78dcae6feaca15fed4b7174668098b00ba6161f8d50151780f18ef168:0

Peer Alias: mainnet.lightningconductor.net

Peer Node Pubkey: 03c436af41160a355fc1ed230a64f6a64bcbd2ae50f12171d1318f9782602be601

Capacity: 500,000

Limbo Balance: 496,530

Local Balance: 496,530

Remote Balance: 0

TRANSACTION 2:

Transaction ID: 8ab4bff8b497ca6044278db7bae9080e03101cb1b9955b5e54018de75ccbd4fb

Channel Point: 21f431e763a12cfc89a8c141e2c431a0c9ffb78a02ceff1053965fc32438d57c:0

Peer Alias: mainnet.lightningconductor.net

Peer Node Pubkey: 03c436af41160a355fc1ed230a64f6a64bcbd2ae50f12171d1318f9782602be601

Capacity: 500,000

Limbo Balance: 455,680

Local Balance: 455,680

Remote Balance: 41,017

**Comment by guggero** [comment]
Looks like the closing transaction never went through. You probably need to force close (`lncli closechannel --force 717143d78dcae6feaca15fed4b7174668098b00ba6161f8d50151780f18ef168 0`) and then wait until the time lock expires (in this case should be around 144 blocks, so 1 day).

**Reply by deserteagle2303** [reply]
How do I enter that command?  through lightning terminal or can i use thunderhub?

**Reply by guggero** [reply]
Maybe there's a button in Thunderhub that says "force close" somewhere? Don't know Thunderhub really, sorry. What kind of system are you running? Consult the documentation of your node software (RaspiBlitz, Umbrel, myNode???) to find out how to enter the command line.

**Reply by deserteagle2303** [reply]
I am running Umbrel
I will check the docs to see how to enter the command line.  Thank you so much

**Reply by LNOrville** [reply]
Umbrel has an app called lightning shell where you can enter the lncli commands

**Reply by deserteagle2303** [reply]
Thank you.  I will install that.

**Reply by deserteagle2303** [reply]
@guggero and @LNOrville , I want to thank both of you for your help.  Finally, the closure worked and the funds have been returned to my wallet.  I am really new to the lightning network, what resources can you both point me to in order to properly setup things so I am actually contributing to the network instead of having my funds  stuck in limbo?  Thank you so much
